### ISSUE TITLE
Submission not storing.

### ISSUE THREAD

**Issue by ashishsurya** [hkirat/peetcode#11]

**Comment by Arghyahub** [comment]
Hey @ashishsurya 
The devops part of the project is not done yet. It will soon be completed in the future until then you can't actually run the code

But if you open the console and run submit, you will see a Accept or Rejected status for now 😊

**Comment by ashishsurya** [comment]
> Hey @ashishsurya 
> The devops part of the project is not done yet. It will soon be completed in the future until then you can't actually run the code
> 
> But if you open the console and run submit, you will see a Accept or Rejected status for now 😊

The random submission are also not storing in the server

**Comment by Arghyahub** [comment]
You are correct, There's no frontend button to check submissions and no logic written to store submission history.
I would surely work on that🙂

**Comment by ashishsurya** [comment]
Logic is written in backend to store random submissions

https://github.com/hkirat/peetcode/blob/ebc840cf07247345c3d7e01dd18de6546d347063/server/index.js#L145

**Comment by Sivayogeith** [comment]
> You are correct, There's no frontend button to check submissions and no logic written to store submission history. I would surely work on that🙂

There is logic for both of it in my PR #7

**Comment by Arghyahub** [comment]
> There is logic for both of it in my PR #7

Great Job⭐
